apiserver/pkg/client/cluster.go

```diff
@@ -25,7 +26,7 @@ func (cc RayClusterClient) RayClusterClient(namespace string) rayiov1alpha1.RayC
 func NewRayClusterClientOrFatal(initConnectionTimeout time.Duration, options util.ClientOptions) ClusterClientInterface {
 	cfg, err := config.GetConfig()
 	if err != nil {
-		glog.Fatalf("Failed to create RayCluster client. Error: %v", err)
+		klog.Fatalf("Failed to create RayCluster client. Error: %v", err)
```

Comment:
Not familiar with `klog` and `glog`. What's the difference between them and why do we choose `klog`?

Reply:
I think we mix usage in different projects earlier. klog is an improved version of glog which has been deprecated.

Reply:
Got it. Thank you for the explanation!